Project/src/Utils/Utils.js

```diff
@@ -64,6 +101,24 @@ export const utils = {
             return 'Unknown Identifier';
         }
     },
+    constructIdentifierFromStringMri: (mri) => {

```

Comment:
nit: change the logic to `switch` instead of `else if`

Reply:
actually switch statement wont be ideal here. The prefix have variable lengths depending on identifier type